packages/svelte/src/lib/createInertiaApp.ts

```diff
@@ -70,21 +72,20 @@ export default async function createInertiaApp({
 
     setup({
       el,
-      App: SvelteApp,
+      App,
       props: {
         initialPage,
         resolveComponent,
       },
     })
-
-    return
   }
 
-  // Svelte types are written for the DOM API and not the SSR API.
-  const { html, head, css } = (SSR as any).render({ id, initialPage })
+  if (isServer) {
```

Comment:
Instead of the additional `isServer` check here, should we just move this above the check above and remove the second check?

Reply:
The condition used to be present, it was only removed when the typescript types were merged : https://github.com/inertiajs/inertia/commit/c01b56e5bc094fd5e5e3d1200590969208ba1bf2

We indeed can avoid the condition, however this is more alike of how vue2, vue3 and react adapter works. It's also very clear that the below code is only intended for SSR only

Reply:
The other adapters do the `isServer` check in order to run `setupProgress`, for my money I think it's more readable if we move the `isServer` check up which makes the `if` block shorter and easier to parse, then just return otherwise.

Reply:
Done!